src/components/AssignmentReminderModal/emailTemplate.js

```diff
@@ -0,0 +1,12 @@
+const emailTemplate = `This is a reminder email that your learning manager has provided you with a access code to take a course at edX. You have redeemed this code {REDEEMED_OFFER_COUNT} of times out of {TOTAL_OFFER_COUNT} number of available course redemptions.
```

Comment:
Also ` ... number of available course redemptions.` can be `... number of available coupon redemptions` ???